utils/k6_helpers.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+from utils.accounts import EthAccounts
+from utils.faucet import Faucet
+from utils.web3client import NeonChainWeb3Client
+
+def k6_prepare_accounts(network, users, balance, bank_account):
+    print("Setting k6 envs...")
+    config = k6_set_envs(network, "envs.json", users, balance, bank_account)
```

Comment:
you can use class NetworkManager which was created for reading network params from this file

Reply:
done